src/templates/partials/custom-buttons.html

```diff
@@ -1,5 +1,5 @@
 <div class="custom-buttons">
-    <ul class="social-likes" data-url="http://jqapi.ru" data-title="jQuery API Cheatsheet">
+    <ul class="social-likes" data-url="https://futurecolors.github.io/jqapi.ru/" data-title="jQuery API Cheat Sheet">
```

Comment:
I see, seems like they use a folder with the project name of organization domain by default (when CNAME file is not present). From [here](https://help.github.com/articles/custom-domain-redirects-for-github-pages-sites/). How about I update my PR to have `CNAME` file with "custom" URL as "https://futurecolors.github.io/jqapi.ru" - hope this works and we will be able to use community-friendly `github.io` domain name instead of `futurecolors.ru`.